styles/components/Search.module.scss

```diff
@@ -0,0 +1,44 @@
+@import "../utils/mixins";
+
+.searchContainer {
+  background-color: var(--nypl-colors-ui-gray-x-light-cool);
+  padding: var(--nypl-space-l) 0 var(--nypl-space-xs);
+}
+
+.searchFormContainer {
+  @include maxWidth;
+
+  @include media(var(--nypl-breakpoint-xl)) {
```

Comment:
This doesn't work. CSS var arguments don't actually get passed to mixins at compile time.

Reply:
Now I understand why the hardcoded values were in the breakpoints scss file on discovery-front-end. I restored it, but let me know if I should take a different approach.